root/edit/details/AddRemoveAlias.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+/* eslint-disable no-negated-condition */
+import React from 'react';
+
+import bracketed from '../../static/scripts/common/utility/bracketed';
+import formatEntityTypeName from '../../static/scripts/common/utility/formatEntityTypeName';
+import formatDate from '../../static/scripts/common/utility/formatDate';
+import isDateEmpty from '../../static/scripts/common/utility/isDateEmpty';
+import DescriptiveLink from '../../static/scripts/common/components/DescriptiveLink';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import isolateText from '../../static/scripts/common/utility/isolateText';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type AliasEntityTypeT = $ElementType<AliasEntityTypeT, 'entityType'>;
+
+type DisplayDataT = {
+  alias: string,
+  begin_date: PartialDateT,
+  end_date: PartialDateT,
+  ended: boolean,
+  entity_type: AliasEntityTypeT,
+  locale: string,
+  primary_for_locale: boolean,
+  sort_name: string,
+  type: AliasT,
+};
+
+type Props = {
+  ...EditT,
+  display_data: DisplayDataT,
+};
+
+const AddRemoveAlias = ({edit}: {edit: Props}) => {
+  const display = edit.display_data;
+  const entityType = display.entity_type;
+  const entity = display[entityType];
+  return (
+    <table className={`details ${edit.edit_kind}-${entityType}-alias`}>
+      <tr>
+        <th>{addColon(formatEntityTypeName(entityType))}</th>
```

Comment:
```suggestion
        <th>{addColonText(formatEntityTypeName(entityType))}</th>
```

Same for the uses of `addColon` below.

Reply:
addColonText was used so suppress flow errors, then return of formatEntityTypeName is Str so I used addColon, I dont think there is much difference in both other than PropTypes

Reply:
`addColonText` is guaranteed (both at run time and by the type system) to return a string, so it should be used where a string is expected.

Reply:
To be clear, it wouldn't cause any particular issue at runtime if `addColon` returned a React element here, since it would display fine inside a `<th>` element. But it would be unexpected, since our expectation is for a plain string to be displayed. :)

(The parser is also (slightly) more efficient if you use `addColonText` since it doesn't need to look for HTML, but that's a negligible difference.)